keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ResourceHistoryService.kt

```diff
@@ -136,9 +130,17 @@ class ResourceHistoryService(
   }
 
   /**
-   * Checks last 10 events for flapping between only ResourceActuationLaunched and ResourceDeltaDetected
+   * Returns true if a resource has been vetoed, or if the last 10 events are only
+   * ResourceActuationLaunched or ResourceDeltaDetected events.
+   *
+   * Even though we could return the status of vetoed we think that unhappy is more clear to users
+   * and doesn't leak implementation terminology that might not be helpful to users.
```

Comment:
I totally agree. I'm not sure what wording/icon we were using to surface that status on the UI, but as a user I'd have no clue what "vetoed" means.